scalafix/rules/src/main/scala/fix/Scalacollectioncompat_newcollections.scala

```diff
@@ -278,9 +484,8 @@ case class Scalacollectioncompat_newcollections(index: SemanticdbIndex)
       replaceSetMapPlus2(ctx) +
       replaceMutSetMapPlus(ctx) +
       replaceMutMapUpdated(ctx) +
-      replaceIterableSameElements(ctx) +
       replaceArrayBuilderMake(ctx) +
-      replaceMapMapValues(ctx)
-
+      replaceMapMapValues(ctx) +
+      replaceIterableSameElements(ctx)
```

Comment:
Why do we need to move that one in last position?

Reply:
just a rebase mistake, nothing important. However, we need to run CanBuildFrom before ToList.